maskrcnn_benchmark/data/datasets/voc_eval.py

```diff
@@ -0,0 +1,186 @@
+# A modification version from chainercv repository.
+# (See https://github.com/chainer/chainercv/blob/master/chainercv/evaluations/eval_detection_voc.py)
+from __future__ import division
```

Comment:
I wonder if this file should live in `datasets`, or in a separate folder dedicated to inference. Thoughts?

Reply:
In fact, `inference.py` looks a little chaotic.
I think  `inference.py` should only do the predication, then pass results and call different evaluatation methods according to the dataset_name. Like you mentioned before.

The location of 'different evaluatation methods', I think, can be:
```
datasets
  -evaluatation
    -coco
      ...
      ...
    -voc
     ...
     ...
   coco.py
   voc.py
```

What do you think?

Reply:
I agree with you, let's keep this structure that you proposed.

Reply:
I've make a new commit. Thanks for your review again.